airbyte-integrations/connectors/source-braintree/integration_tests/configured_catalog.json

```diff
@@ -0,0 +1,81 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "customer_stream",
+        "json_schema": {},
+        "supported_sync_modes": ["full_refresh"],
```

Comment:
I look at the code and don't understand why your `customer_stream` stream can be Incremental, but in the catalog you indicate it simply as `full_refresh`?

Reply:
Isn't it ignored by acceptance test anyway? What point of those changes to integration_tests/configured_catalog.json ?

Reply:
Updated